src/routes/quickstarts/nuxt/setup.mdx

```diff
@@ -0,0 +1,103 @@
+# Setup a Nuxt app with Mirage for Development
+
+Mock your Nuxt app's networking code so you can develop your Nuxt app without any backend services.
+
+> This is a quickstart guide for people familiar with Mirage. If you're brand new to Mirage, take a look at the [Overview](/docs/getting-started/overview).
+
+## Step 1: Install Mirage
+
+First, make sure you have Mirage installed:
+
+```bash
+# Using npm
+npm install --save-dev miragejs
+
+# Using Yarn
+yarn add --dev miragejs
+```
+
+## Step 2: Define your server
+
+Under the plugins Nuxt folder, create a new `mirage.js` file and define your mock server.
+
+Here's a basic example:
+
+```js
+// src/plugins/mirage/mirage.js
+import { Server, Model } from "miragejs"
+
+export function makeServer({ environment = "development" } = {}) {
+  let server = new Server({
+    environment,
+
+    models: {
+      user: Model,
+    },
+
+    seeds(server) {
+      server.create("user", { name: "Bob" })
+      server.create("user", { name: "Alice" })
+    },
+
+    routes() {
+      this.namespace = "api"
+
+      this.get("/users", schema => {
+        return schema.users.all()
+      })
+    },
+  })
+
+  return server
+}
+```
```

Comment:
One can hook into the plugins by exporting the above function wrapped in a default function
```suggestion
export default ({ app }) => {
  makeServer({ environment: 'development' })
}
```
```
Note: The code block backtick is showing deleted in the above review since GitHub couldn't parse multiple backticks.